nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListenTCPRecord.java

```diff
@@ -87,14 +93,28 @@
         "If the read times out, or if any other error is encountered when reading, the connection will be closed, and any records " +
         "read up to that point will be handled according to the configured Read Error Strategy (Discard or Transfer). In cases where " +
         "clients are keeping a connection open, the concurrent tasks for the processor should be adjusted to match the Max Number of " +
-        "TCP Connections allowed, so that there is a task processing each connection.")
+        "TCP Connections allowed, so that there is a task processing each connection. " +
+        "The processor can be configured to use an SSL Context Service to only allow secure connections. " +
+        "When a client connects to the processor using secure connection, the Distinguished Names of the client certificate's " +
```

Comment:
The client certificate is only available for mutual TLS, and it would not be available when client authenticated is disabled. Recommend the following wording for clarity:
```suggestion
        "When connected clients present certificates for mutual TLS authentication, the Distinguished Names of the client certificate's " +
```